src/modules/app/fetchData.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import type { Dispatch } from 'redux-first-router'
+import type { Dispatch } from 'redux-first-router/dist/flow-types.js.flow'
```

Comment:
can can write just `'redux-first-router/dist/flow-types'`

Is there no other way we can import this? Maybe we should report this as a bug to the router.

Reply:
I don't know why some types got removed from the redux-first-router index.js, but I don't think it was an accident.

Reply:
Hm that was the point I stopped to do this task last time. The upgrade did not seem finished when types are missing.

Reply:
Types aren't missing, the types are the redux types, as pointed out by @Schedulaar.